node/service/src/chain_spec/base.rs

```diff
@@ -230,7 +230,7 @@ fn testnet_genesis(
 			],
 			metadata: vec![
 				// id, name, symbol, decimals
-				(LOCAL_ASSET_ID, "BASE".into(), "xUSD".into(), 10),
```

Comment:
I think this can be dropped.
See https://github.com/paritytech/trappist/pull/81

Reply:
OK will remove it here as well, thanks Bernardo!